src/BotPlutusInterface/Files.hs

```diff
@@ -167,50 +173,82 @@ readPrivateKeys ::
   forall (w :: Type) (effs :: [Type -> Type]).
   Member (PABEffect w) effs =>
   PABConfig ->
-  Eff effs (Either Text (Map PubKeyHash PrivateKey))
+  Eff effs (Either Text (Map PubKeyHash DummyPrivKey))
 readPrivateKeys pabConf = do
   files <- listDirectory @w $ Text.unpack pabConf.pcSigningKeyFileDir
+  let vKeyFiles =
+        map (\filename -> Text.unpack pabConf.pcSigningKeyFileDir ++ "/" ++ filename) $
+          filter ("vkey" `isExtensionOf`) files
   let sKeyFiles =
         map (\filename -> Text.unpack pabConf.pcSigningKeyFileDir ++ "/" ++ filename) $
           filter ("skey" `isExtensionOf`) files
```

Comment:
I feel like these could be combined in some way, are the files always either skey or vkey? if so, a partition could be used. either way, the map function could be applied before the filter to avoid writing it twice

Reply:
Fixed.